src/WormholeJoin.sol

```diff
@@ -204,17 +203,15 @@ contract WormholeJoin {
     **/
     function withdrawPending(WormholeGUID calldata wormholeGUID, uint256 maxFee) external {
         require(wormholeGUID.operator == msg.sender, "WormholeJoin/sender-not-operator");
-        _withdraw(wormholeGUID, maxFee);
+        _withdraw(wormholeGUID, getGUIDHash(wormholeGUID), maxFee);
     }
 
     /**
     * @dev External authed function that repays debt with DAI coming from the bridges
     * @param sourceDomain domain where the DAI is coming from
     * @param batchedDaiToFlush Amount of DAI that is being processed for repayment
     **/
-    function settle(bytes32 sourceDomain, uint256 batchedDaiToFlush) external auth {
-        // TODO: define if we want to change to pull model instead of push
-        // (this function expects to have received batchedDaiToFlush erc20 DAI before settle being called)
+    function settle(bytes32 sourceDomain, uint256 batchedDaiToFlush) external {
```

Comment:
The @dev comment needs updating (External authed function -> External function)

Reply:
Done!